net/haproxy/src/opnsense/mvc/app/views/OPNsense/HAProxy/index.volt

```diff
@@ -308,50 +361,155 @@ POSSIBILITY OF SUCH DAMAGE.
             </ul>
         </li>
-    {% else %}
+        {% else %}
         {# Standard Tab #}
-        <li {% if mainForm['activetab']|default("") == tab[0] %} class="active" {% endif %}>
+        <li>
                 <a data-toggle="tab" href="#tab_{{tab[0]}}">
                     <b>{{tab[1]}}</b>
                 </a>
         </li>
-    {% endif %}
-{% endfor %}
-    {# add custom content #}
-    <li><a data-toggle="tab" href="#frontends"><b>{{ lang._('Frontends') }}</b></a></li>
-    <li><a data-toggle="tab" href="#backends"><b>{{ lang._('Backends') }}</b></a></li>
-    <li><a data-toggle="tab" href="#servers"><b>{{ lang._('Servers') }}</b></a></li>
-    <li><a data-toggle="tab" href="#healthchecks"><b>{{ lang._('Health Checks') }}</b></a></li>
-    <li><a data-toggle="tab" href="#actions"><b>{{ lang._('Actions') }}</b></a></li>
-    <li><a data-toggle="tab" href="#acls"><b>{{ lang._('ACLs') }}</b></a></li>
-    <li><a data-toggle="tab" href="#luas"><b>{{ lang._('Lua Scripts') }}</b></a></li>
-    <li><a data-toggle="tab" href="#errorfiles"><b>{{ lang._('Error Files') }}</b></a></li>
+        {% endif %}
+    {% endfor %}
+
+    <li role="presentation" class="dropdown">
+        <a data-toggle="dropdown" href="#" class="dropdown-toggle pull-right visible-lg-inline-block visible-md-inline-block visible-xs-inline-block visible-sm-inline-block" role="button" style="border-left: 1px dashed lightgray;">
+            <b><span class="caret"></span></b>
+        </a>
+        <a data-toggle="tab" href="#subtab_haproxy-advanced-introduction" class="visible-lg-inline-block visible-md-inline-block visible-xs-inline-block visible-sm-inline-block" style="border-right:0px;"><b>{{ lang._('Advanced') }}</b></a>
+        <ul class="dropdown-menu" role="menu">
+            <li><a data-toggle="tab" href="#subtab_haproxy-advanced-introduction"><i class="fa fa-check-square"></i> {{ lang._('Introduction') }}</a></li>
+            <li><a data-toggle="tab" href="#errorfiles"><i class="fa fa-check-square"></i> {{ lang._('Error Messages') }}</a></li>
+            <li><a data-toggle="tab" href="#luas"><i class="fa fa-check-square"></i> {{ lang._('Lua Scripts') }}</a></li>
+        </ul>
+    </li>
 </ul>
 
 <div class="content-box tab-content">
+    <div id="introduction" class="tab-pane fade in active">
+        <div class="col-md-12">
+            <h1>Quick Start Guide</h1>
+            <p>{{ lang._('Welcome to the HAProxy plugin! This plugin is designed to offer all the features and flexibility HAProxy is famous for. If you are using HAProxy for the first time, please take some time to get familiar with it. The following information should help you to get started.')}}</p>
+            <p>{{ lang._('Note that you should configure everything in the following order:') }}</p>
+            <ul>
+              <li>{{ lang._('Add %sReal Servers:%s All physical or virtual servers that you want HAProxy to use should be added here.') | format('<b>', '</b>') }}</li>
```

Comment:
"Real" is introduced here but other pages only use "Servers". You could go to the extreme with this an call stuff "Servers", "Pools", "Services" ;)

Also, it's not perfectly clear what this means, maybe better to reword like "that you want HAPproxy to use" to "that HAProxy to serve to" ?

Reply:
@fichtner OK to replace "that you want HAProxy to use should be added here" with "that HAProxy should use for loadbalancing or proxying must be added here"?

Reply:
yes and no, the direction should be clearer e.g. "load balance between or proxy to"